packages/ckeditor5-package-generator/lib/templates/ts/package.json

```
@@ -80,14 +81,16 @@
     "typescript": "^4.7.4"
   },
   "scripts": {
+    "build": "tsc -p ./tsconfig.release.json",
     "dll:build": "ckeditor5-package-tools dll:build",
     "dll:serve": "http-server ./ -o sample/dll.html",
     "lint": "eslint \"**/*.{js,ts}\" --quiet --ignore-pattern \"build/\"",
     "start": "ckeditor5-package-tools start",
     "stylelint": "stylelint --quiet --allow-empty-input 'theme/**/*.css'",
     "test": "ckeditor5-package-tools test",
     "prepare": "<%= program %> run dll:build",
-    "prepublishOnly": "<%= program %> run dll:build",
+    "prepublishOnly": "<%= program %> run dll:build && <%= program %> run build && ckeditor5-package-tools updatePkgJsonMain js",
+    "postpublish": "npx rimraf \"src/**/*.@(js|d.ts)\" && ckeditor5-package-tools updatePkgJsonMain ts",
```

```suggestion
    "ts:clear": "npx rimraf \"src/**/*.@(js|d.ts)\" && ckeditor5-package-tools export-package-as-typescript",
```

I changed two things:

1. The name of an exported script. If you read the last post-mortem regarding CKEditor 5 release, you can understand why I suggest it. Automatization is good, but we need to know the limits. When it comes to releasing, it would be better to clear things manually once everything is done.
2. As suggested in the `prepublishOnly` hook, unify these scripts.

---

I don't think that clearing built files and flipping the `main` field back has potential to cause problems, event in case of broken release process. If we still want to disable it, we could perhaps prompt the user to remind him that he has to call this script manually, otherwise his project wont work properly.

---

I agree. Let's leave it as it is now.